### PR TITLE
fix(suite-native): use 20px border radius instead of 24px

### PR DESCRIPTION
**Pull request by yanascz**
20px border radius is the biggest one available according to design.

## Related Issue

#11921

### CODE REVIEW

File: suite-native/module-authorize-device/src/components/passphrase/PassphraseForm.tsx [hunk 1]
```diff
@@ -44,7 +44,6 @@ type PassphraseFormProps = {
 
 const formStyle = prepareNativeStyle(utils => ({
     backgroundColor: utils.colors.backgroundSurfaceElevation1,
-    borderRadius: utils.borders.radii.r24,
```
Comment (yanascz) [inline suite-native/module-authorize-device/src/components/passphrase/PassphraseForm.tsx:47]: There doesn't seem to be any visible border so it doesn't make sense here. Correct me if I'm wrong. 🙏

Comment (Nodonisko) [reply suite-native/module-authorize-device/src/components/passphrase/PassphraseForm.tsx:47]: You don't need border to make something rounded, background is enough

Comment (yanascz) [reply suite-native/module-authorize-device/src/components/passphrase/PassphraseForm.tsx:47]: I understand that, I'm saying there's no visible effect of the radius change. I believe even the background colour is redundant here.

Comment (PeKne) [reply suite-native/module-authorize-device/src/components/passphrase/PassphraseForm.tsx:47]: seems like that

Comment (yanascz) [reply suite-native/module-authorize-device/src/components/passphrase/PassphraseForm.tsx:47]: This is the area we're talking about:
image

Comment (yanascz) [reply suite-native/module-authorize-device/src/components/passphrase/PassphraseForm.tsx:47]: Removed the form style altogether. 1412384

**Review by PeKne — APPROVED** [review]

**Pull request merged**